src/pages/HomePage/HomePage.tsx

```diff
@@ -1,13 +1,24 @@
+import { Box } from '@material-ui/core';
 import * as React from 'react';
-
-import Button from '../../components/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import ThreadsList from '../../components/ThreadsList';
 
 const HomePage: React.FC = () => {
-    const [ s, uS] = React.useState(0);
+    //const [ s, uS] = React.useState(0);
 
     return (
-        <Button onClick={() => uS(p => p+1)} label={`You clicked ${s} times`} />
+        <React.Fragment>
```

Comment:
Można po prostu
```
<>
</>
```

`<>` to skrócona wersja `React.Fragment`

Reply:
👍